.github/workflows/build-container.yml

```
@@ -223,13 +223,29 @@ jobs:
         run: |
           make demo OPTS=-T NVMEOF_CONTAINER_NAME="ceph-nvmeof_nvmeof_1"
 
-      - name: Get subsystems
+      - name: List resources
         run: |
           # https://github.com/actions/toolkit/issues/766
           shopt -s expand_aliases
           eval $(make alias)
-          nvmeof-cli get_subsystems
-          nvmeof-cli-ipv6 get_subsystems
+          cephnvmf subsystem list
+          subs=$(cephnvmf --output stdio --format json subsystem list | grep nqn | sed 's/"nqn": "//' | sed 's/",$//')
+          for sub in $subs
+          do
+              cephnvmf namespace list --subsystem $sub
+              cephnvmf listener list --subsystem $sub
+              cephnvmf host list --subsystem $sub
+          done
+
+      - name: Verify no HA
+        run: |
+          shopt -s expand_aliases
+          eval $(make alias)
+          ha_status=$(cephnvmf --output stdio --format json subsystem list | grep "enable_ha" | grep "true" | tr -d '\n\r')
```

`jq` is a great tool to [parse JSON data](https://www.devtoolsdaily.com/jq_playground/). Something like this:
```suggestion
          ha_status=$(cephnvmf --output stdio --format json subsystem list | jq '.enable_ha'
```